src/dialogsformaction.cpp

```diff
@@ -83,28 +83,18 @@ bool DialogsFormAction::process_operation(Operation op,
 {
 	switch (op) {
 	case OP_OPEN: {
-		std::string dialogposname = f.get("dialogs_pos");
-		if (dialogposname.length() > 0) {
-			v->set_current_formaction(utils::to_u(dialogposname));
-		} else {
-			v->show_error(_("No item selected!"));
-		}
+		unsigned int pos = dialogs_list.get_position();
```

Comment:
Can this be `const`?

Same for all other variables where you changed the initializer.